Drive/rover_drive_system.hpp

```diff
@@ -81,8 +83,8 @@ class RoverDriveSystem
       // TODO: Account for heartbeat procedure i.e. "messageCount": 132
       sscanf(
           response.c_str(),
-          R"({ "is_operational": %d, "drive_mode": "%c", "speed": %d, "angle": %d })",
-          &mc_data.is_operational, &mc_data.drive_mode, &mc_data.speed,
+          R"({ "heartbeatCount": %d, is_operational": %d, "drive_mode": "%c", "speed": %d, "angle": %d })",
```

Comment:
I think we should make this match the other variables. e.g. `heartbeat_count`.
Need to modify this on the web server side as well.